### ISSUE TITLE
Can't scroll the webview when touching on the native ad in version 4.22.2

### ISSUE THREAD

**Issue by JRCuellar** [floatinghotpot/cordova-plugin-facebookads#62]
Hi,

Firts, excuse me, my english isn't very good.

I've app in Ionic 3 that used this plugin in version 4.7.4. This worked ok. This week I updated to version 4.22.2 and all ok except that I can't scroll when touching on the native ad. Are you sure that is working ok?

I've this problem only when I do scroll inside native ad. When I touch outside native ad all is ok.

**Comment by JRCuellar** [comment]
Hi and excuse me for my english


Now I've resolve this problem. In the code of plugin "/plugins/cordova-plugin-facebookads/src/android/FacebookAdPlugin.java", I've changed in line 165 the order of two lines:

layout.addView(unit.view, new RelativeLayout.LayoutParams(unit.w, unit.h));
layout.addView(unit.tracking, new RelativeLayout.LayoutParams(unit.w, unit.h));

to:

**layout.addView(unit.tracking, new RelativeLayout.LayoutParams(unit.w, unit.h));
layout.addView(unit.view, new RelativeLayout.LayoutParams(unit.w, unit.h));**

The view "tracking" was created over the view "view" and this don't run the event of the view "view".

I hope help you my solution.

**Comment by floatinghotpot** [comment]
@JRCuellar , thank you for the amazing advice. 

Just one more thing to double confirm, after exchange the two lines of code, can you trigger the ad when click the area?

**Comment by JRCuellar** [comment]
@floatinghotpot , yes I can click the area in ad. Actually I install the version 4.23.0 of plugin and I can with this change. I test it in four devices. Samsung Galaxy Note 2, LG Optimus G, Samsung J3 and Samsung Galaxy S7 Edge. Android 4.4, Android 5.1, Android 6. 


**Ionic Info**

global packages:

    @ionic/cli-utils : 1.4.0
    Cordova CLI      : 6.5.0 
    Ionic CLI        : 3.4.0

local packages:

    @ionic/app-scripts              : 1.3.7
    @ionic/cli-plugin-cordova       : 1.4.0
    @ionic/cli-plugin-ionic-angular : 1.3.1
    Cordova Platforms               : android 6.1.2 ios 4.4.0
    Ionic Framework                 : ionic-angular 3.2.1

System:

    Node       : v7.0.0
    OS         : Linux 3.19
    Xcode      : not installed
    ios-deploy : not installed
    ios-sim    : not installed
    npm        : 4.3.0 


**Plugin list**

com-sarriaroman-photoviewer 1.1.10 "PhotoViewer"
cordova-facebook-audnet-sdk 4.23.0 "Facebook AudienceNetwork SDK for Cordova"
cordova-plugin-admobpro 2.28.3 "AdMob Plugin Pro"
cordova-plugin-app-version 0.1.9 "AppVersion"
cordova-plugin-console 1.0.5 "Console"
cordova-plugin-device 1.1.4 "Device"
cordova-plugin-extension 1.5.1 "Cordova Plugin Extension"
cordova-plugin-facebook4 1.8.0 "Facebook Connect"
cordova-plugin-facebookads 4.23.0 "FacebookAds"
cordova-plugin-google-analytics 1.8.2 "Google Universal Analytics Plugin"
cordova-plugin-googleplus 5.1.1 "Google SignIn"
cordova-plugin-inappbrowser 1.7.1 "InAppBrowser"
cordova-plugin-network-information 1.3.3 "Network Information"
cordova-plugin-splashscreen 4.0.3 "Splashscreen"
cordova-plugin-statusbar 2.2.2 "StatusBar"
cordova-plugin-whitelist 1.3.1 "Whitelist"
cordova-plugin-x-socialsharing 5.1.8 "SocialSharing"
cordova-sqlite-storage 2.0.4 "Cordova sqlite storage plugin"
es6-promise-plugin 4.1.0 "Promise"
ionic-plugin-keyboard 2.2.1 "Keyboard"
phonegap-plugin-push 1.10.3 "PushPlugin"

**Comment by floatinghotpot** [comment]
I've merged your modification, thanks a lot!
